editor.planx.uk/src/@planx/components/ContactInput/Public.tsx

```diff
@@ -0,0 +1,156 @@
+import Card from "@planx/components/shared/Preview/Card";
+import QuestionHeader from "@planx/components/shared/Preview/QuestionHeader";
+import type { PublicProps } from "@planx/components/ui";
+import { useFormik } from "formik";
+import React from "react";
+import Input from "ui/Input";
+import InputLabel from "ui/InputLabel";
+import InputRowItem from "ui/InputRowItem";
+
+import { ERROR_MESSAGE } from "../shared/constants";
+import { getPreviouslySubmittedData, makeData } from "../shared/utils";
+import type { Contact, ContactInput } from "./model";
+import { userDataSchema } from "./model";
+
+export type Props = PublicProps<ContactInput, Contact>;
+
+interface FormProps {
+  title: string;
+  firstName: string;
+  lastName: string;
+  organisation: string;
+  phone: string;
+  email: string;
+  "name.first"?: string;
+  "name.last"?: string;
+  "company.name"?: string;
+  "phone.primary"?: string;
+}
+
+export default function ContactInputComponent(props: Props): FCReturn {
+  const formik = useFormik<FormProps>({
+    initialValues: getPreviouslySubmittedData(props) ?? {
+      title: "",
+      firstName: "",
+      lastName: "",
+      organisation: "",
+      phone: "",
+      email: "",
+    },
+    onSubmit: (values) => {
+      // map internal props to expected granular passport keys before submitting
+      values["name.first"] = values.firstName;
+      values["name.last"] = values.lastName;
+      values["company.name"] = values.organisation;
+      values["phone.primary"] = values.phone;
+
+      // update passport on submit
+      props.handleSubmit?.(makeData(props, values));
+    },
+    validateOnBlur: false,
+    validateOnChange: false,
+    validationSchema: userDataSchema,
+  });
+
+  return (
+    <Card handleSubmit={formik.handleSubmit}>
+      <QuestionHeader
+        title={props.title}
+        description={props.description}
+        info={props.info}
+        policyRef={props.policyRef}
+        howMeasured={props.howMeasured}
+      />
+      <InputLabel label="Title">
+        <InputRowItem width="40%">
+          <Input
+            name="title"
+            value={formik.values.title}
+            bordered
+            errorMessage={formik.errors.title}
+            onChange={formik.handleChange}
+            id={`${props.id}-title`}
+            inputProps={{
+              "aria-describedby": formik.errors.title
+                ? `${ERROR_MESSAGE}-${props.id}-title`
+                : "",
+            }}
+          />
+        </InputRowItem>
+      </InputLabel>
```

Comment:
perhaps interesting to note that GovUK suggests avoiding asking for a person's title, or at minimum making it optional: https://design-system.service.gov.uk/patterns/names/

i remember previous conversations in planning about this where partners were very resistant to the idea of not collecting this field because officers are used to referencing his information when making followup phone calls, but might be worth raising again when this goes to UAT